gomock/controller.go

```diff
@@ -80,7 +83,7 @@ func NewController(t TestReporter) *Controller {
 	}
 }
 
-func (ctrl *Controller) RecordCall(receiver interface{}, method string, args ...interface{}) *Call {
+func (ctrl *Controller) RecordCall(receiver interface{}, method string, methodType reflect.Type, args ...interface{}) *Call {
```

Comment:
The breaking change is cleaner and I agree that calling those methods directly doesn't make a lot of sense because the generated mocks are powerful enough.